src/electron.d.ts

```diff
@@ -3842,7 +3842,7 @@ declare namespace Electron {
     /**
      * The window's bounds.
      */
-    Bounds: Rectangle;
```

Comment:
@pbaize it looks like a mistake, needs to be lowercase

Reply:
we should fix it in the runtime docs then, not here.

Reply:
and there too. let's keep this here for now for it can work, otherwise we need to wait and now it won't work without this change

Reply:
feel free to generate types from a feature branch of the runtime